process.go

```diff
@@ -37,18 +37,19 @@ func process(fs fileSystem, filePaths []string) (poFile, error) {
 		if !isSupportedExt(filepath.Ext(filePath)) {
 			continue
 		}
-
-		content, err := fs.readFile(filePath)
-		if err != nil {
-			return poFile, err
+		f, e := os.Open(filePath)
```

Comment:
Revert to `err` to keep in line with standard go practices.

```suggestion
		f, err := os.Open(filePath)
```